examples/step-50/doc/results.dox

```diff
@@ -1,98 +1,215 @@
+  <td align="right">65M</th>
+  <td align="right">0.22</th>
+  <td></th>
+  <td align="right">1.122</th>
+  <td align="right">0.686</th>
+  <td align="right">0.309</th>
+  <td align="right">2.117</th>
+  <td></th>
+  <td align="right">2.171</th>
+  <td align="right">4.862</th>
+  <td align="right">1.660</th>
+  <td align="right">8.693</th>
+  <td></th>
+  <td align="right">1.654</th>
+  <td align="right">0.546</th>
+  <td align="right">1.460</th>
+  <td align="right">3.660</th>
+</tr>
+<tr>
+  <td align="right">7168</th>
+  <td align="right">19</th>
+  <td align="right">256M</th>
+  <td align="right">0.16</th>
+  <td></th>
+  <td align="right">1.214</th>
+  <td align="right">0.893</th>
+  <td align="right">0.521</th>
+  <td align="right">2.628</th>
+  <td></th>
+  <td align="right">2.386</th>
+  <td align="right">7.260</th>
+  <td align="right">2.560</th>
+  <td align="right">12.206</th>
+  <td></th>
+  <td align="right">1.844</th>
+  <td align="right">1.010</th>
+  <td align="right">1.890</th>
+  <td align="right">4.744</th>
+</tr>
 </table>
-This isn't only fewer iterations than in step-5 (each of which
-is, however, much more expensive) but more importantly, the number of
-iterations also grows much more slowly under mesh refinement (again,
-it would be almost constant if the coefficient was constant rather
-than strongly varying as chosen here). This justifies the common
-observation that, whenever possible, multigrid methods should be used
-for second order problems.
+
+The following figure gives the strong scaling for each method for cycle 16
+(32M DoFs) and 19 (256M DoFs) on between 56 to 28672 processors. While the
+matrix-based GMG solver and AMG scale similarly and have a similar time to
+solution, the matrix-free GMG solver scales much better and solves the finer
+problem in roughly the same time as the AMG solver for the coarser mesh with
+only an eighth of the number of unknowns.
+
+<img width="400px" src="https://www.dealii.org/images/steps/developer/step-50-strong-scaling.png" alt="">
 
 
 <h3> Possible extensions </h3>
 
-A close inspection of this program's performance shows that it is mostly
-dominated by matrix-vector operations. step-37 shows one way
-how this can be avoided by working with matrix-free methods.
-
-Another avenue would be to use algebraic multigrid methods. The
-geometric multigrid method used here can at times be a bit awkward to
-implement because it needs all those additional data structures, and
-it becomes even more difficult if the program is to run in %parallel on
-machines coupled through MPI, for example. In that case, it would be
-simpler if one could use a black-box preconditioner that uses some
-sort of multigrid hierarchy for good performance but can figure out
-level matrices and similar things out by itself. Algebraic multigrid
-methods do exactly this, and we will use them in
-step-31 for the solution of a Stokes problem.
+We currently don't have suggestions for possible extensions.
```

Comment:
This is a very sad statement :(

Reply:
It is just perfect as is. :-) Just kidding, we were already thinking about some ideas.

Reply:
Just add the topics you want to discuss at the hackathon. Maybe someone will solve these for you :P